WooCommerce/Classes/ViewRelated/ReusableViews/LeftImageTitleSubtitleToggleTableViewCell.swift

```diff
@@ -58,6 +63,7 @@ class LeftImageTitleSubtitleToggleTableViewCell: UITableViewCell {
         titleLabel?.applyBodyStyle()
         subtitleLabel?.applyFootnoteStyle()
         toggleSwitch?.onTintColor = .primary
+        subtitleLabel.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(subtitleTapped)))
```

Comment:
Yeah if anything we should drop the `?` on the lines above.
If the outlets aren't set when the nib gets loaded, the app will almost certainly crash for other reasons, so typically we would just use them as force-unwrapped since that's the default for what you get when you drag an outlet out from Interface Builder.

Reply:
> Would it make sense to create some sort of configureSubtitleLabel() method here? We could both group .applyFootnoteStyle() and addGestureRecognizer(...) inside it.

I'll take a look at this in a future PR